pygfx/materials/_line.py

```diff
@@ -8,50 +8,59 @@ class LineMaterial(Material):
 
     Parameters
     ----------
+    thickness : float
+        The line thickness expressed in logical pixels. Default 2.0.
+    thickness_space : str
+        The coordinate space in which the thickness is expressed. Default "screen".
     color : Color
         The uniform color of the line (used depending on the ``color_mode``).
-    thickness : float
-        The line thickness expressed in logical pixels.
     color_mode : enum or str
         The mode by which the line is coloured. Default 'auto'.
     map : Texture
         The texture map specifying the color for each texture coordinate. Optional.
     map_interpolation: str
-        The method to interpolate the color map. Either 'nearest' or 'linear'. Default 'linear'.
+        The method to interpolate the color map ('nearest' or 'linear'). Default 'linear'.
+    dash_pattern : tuple
+        The pattern of the dash. Defaults to an empty tuple, meaning no dashing.
```

Comment:
can you explain what valid patterns look like?

Reply:
done

Reply:
Also  #661